src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -142,6 +142,7 @@ struct sdm_sensor_info
     result_type output;
     std::vector<std::string> stats;
     std::string errmsg;
+    const std::string SD_PRESENT_CHECK = "1";
```

Comment:
```suggestion
    constexpr const char* sd_present_check = "1"; 
```
Plus 1 for symbolic constant.
No upper case for variables. https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rl-all-caps
Plus, might as well use compile time constant here since string buys nothing for the comparison done later.

Reply:
Thank you for suggesting this @stsoe. Done.